main.tf

```diff
@@ -5,7 +5,8 @@ locals {
     {
       "storage" : {
         "gcs" : {
-          "bucket" : "${google_storage_bucket.vault.name}"
+          "bucket" : "${google_storage_bucket.vault.name}",
+          "ha_enabled" : "${var.gcs_ha_enabled}"
```

Comment:
Can you remove the HA components and we'll just merge the concurrency improvements.

Reply:
`var.gcs_ha_enabled` defaults to "false" but I can see that adding confusion, as if changing it would enable HA. Remove `ha_enabled` all together or `"ha_enabled" :  "false"`

Reply:
Yeah I’d remove it altogether for now and just have this PR deal is concurrency